api/types/authentication.go

```diff
@@ -516,6 +517,89 @@ func (u *U2F) Check() error {
 			return trace.BadParameter("u2f configuration has an invalid attestation CA: %v", err)
 		}
 	}
+
+	// AppID must be a valid URL.
+	// NOTE: We allow naked AppID's without a scheme for compatibility with WebAuthn
+	//       as we rely on that behaviour to derive WebAuthn configs on-the-fly.
+	//       This is only valid if all facets are also naked however.
+	appIDUrl, err := url.Parse(u.AppID)
```

Comment:
Acronyms are typically all caps in Go identifiers. How about `appIDURL` or just `u`?

Reply:
Changed to appIDURL.